rabbitmq.info.yml

```diff
@@ -2,5 +2,4 @@ name: "RabbitMQ"
 description: "RabbitMQ Integration for Drupal"
 core: "8.x"
 type: "module"
-php: "5.6"
 package: "Performance and scalability"
```

Comment:
const I think was added in php 5.4? I didn't find any compatibilities issues up from php 5.4 (or phpcs says that)